hedy_content.py

```diff
@@ -45,6 +18,27 @@
     'hi': 'HI'
 }
 
+def fill_all_languages(babel):
+  # load all available languages in dict
+  # list_translations of babel does about the same, but without territories.
+  for dirname in babel.translation_directories:
+      if not os.path.isdir(dirname):
+          continue
+
+      for folder in os.listdir(dirname):
+          locale_dir = os.path.join(dirname, folder, 'LC_MESSAGES')
+          if not os.path.isdir(locale_dir):
+              continue
+
+          if filter(lambda x: x.endswith('.mo'), os.listdir(locale_dir)):
+              locale = Locale.parse(folder)
+              ALL_LANGUAGES[folder] = locale.display_name
+
+  # If not other translations are found, add the default locale.
+  if not ALL_LANGUAGES:
+      locale = Locale.parse(babel._default_locale)
+      ALL_LANGUAGES[babel._default_locale] = locale.display_name
```

Comment:
Not sure if this statement is necessary. We use the value of `ALL_LANGUAGES` for alle our translation retrieval, YAMLs and `gettext` translations. If we don't find the translation file, we also don't want to return this language as an option. In this case we still return the `default_locale` as an option, even if no translations are found at all. But as we can't retrieve any translations for this language the website will crash anyway.

Reply:
I think you're right, if the default wasn't found in the for loop, it wil not be found anytime soon.